runtime/lib/mirrors.cc

```diff
@@ -835,12 +835,49 @@ DEFINE_NATIVE_ENTRY(Mirrors_makeLocalClassMirror, 1) {
                            Object::null_instance());
 }
 
-
 DEFINE_NATIVE_ENTRY(Mirrors_makeLocalTypeMirror, 1) {
   GET_NON_NULL_NATIVE_ARGUMENT(AbstractType, type, arguments->NativeArgAt(0));
   return CreateTypeMirror(type);
 }
 
+DEFINE_NATIVE_ENTRY(Mirrors_makeLocalTypeMirrorWithTypeArguments, 2) {
+  GET_NON_NULL_NATIVE_ARGUMENT(AbstractType, type, arguments->NativeArgAt(0));
+  GET_NON_NULL_NATIVE_ARGUMENT(Array, args, arguments->NativeArgAt(1));
+
+  const Class& clz = Class::Handle(type.type_class());
+  if (!clz.IsGeneric()) {
+    const Array& errorArgs = Array::Handle(Array::New(3));
+    errorArgs.SetAt(0, type);
+    errorArgs.SetAt(1, String::Handle(String::New("key")));
+    errorArgs.SetAt(2, String::Handle(String::New("Type must be a generic class")));
```

Comment:
Thanks! This is interesting, I've been trying to follow examples in existing code and everywhere I could find there was no terminating `.`.
Examples: [1](https://github.com/dart-lang/sdk/blob/master/runtime/lib/mirrors_impl.dart#L1074), [2](https://github.com/dart-lang/sdk/blob/master/runtime/lib/mirrors_impl.dart#L1151), [3](https://github.com/dart-lang/sdk/blob/master/runtime/lib/mirrors_impl.dart#L1560).

Curious if this is reflected in a style guide somewhere or just a personal preference thing?
(I'd personally prefer to terminate with a `.` as well, but staying consistent is a higher priority).